docs/api/covidcast-signals/nssp.md

```diff
@@ -0,0 +1,104 @@
+
+[The NSSP site participation map](https://www.cdc.gov/nssp/media/images/2024/04/Participation-with-date.png) shows counties containing at least one reporting ED between January 2023 and April 2024.
+California, Colorado, Missouri, Oklahoma, and Virginia have the most noticeable gaps in coverage, with many component counties having either no eligible EDs or having no recently reported data in NSSP. However, most states have some counties that are absent.
+
+NSSP does not report county-level data for all counties with reporting EDs; some reporting EDs are only included in state-level values.
+
+The following states report no data through NSSP at the county level: CA, WA, AK, AZ, AL, CO, SD, ND, MO, AR, FL, OH, NH, CT, NJ.
+
+South Dakota, Missouri, and territories report no data through NSSP at the state level.
+
+
+## Lag and Backfill
+
+The weekly signal is reported on Friday mornings, adding data from the prior week.
+For example, on Friday, 2024-04-19, the source added new data from the week ending 2024-04-13.
+
+This data source has frequent backfill, primarily arising from newly included EDs. When a new facility joins the reporting network, its historical data is added to the dataset, resulting in changes to historical values for every geographic level that ED is part of (county through nation). Because of this, the broadest geographic levels are more likely to be revised.
+
+In previous revisions, we have noted changes to values dating back about 2 years.
+
+
+## Limitations
+
+There is substantial missingness at the county level. This tends to impact more rural and lower population locations. See the [missingness section](#missingness) for more information.
+
+Not all counties contain reporting EDs, including in states where NSSP reports state-level data.
+
+NSSP notes that not every patient entering an ED is tested for the conditions of interest, so the data may undercount total cases and as a result percent visits may be biased downward.
+
+Our [geographic weighting approach](#geographic-weighting) assumes that the number of ED visits is proportional to the overall population of a county. However, in reality, there are various factors that could affect the accuracy of this assumption.
+
+For example, we might expect denser, more urban counties to have 1) more and larger EDs and 2) easier access to EDs. The first factor may mean that residents of rural counties are more likely to go to EDs in urban counties. The second factor may increase the total number of ED visits that someone living in an urban county will make, that is, the average urban resident may make more ED visits than the average rural resident over a given period of time.
+
+As a result, total ED visits per capita in rural counties may be lower than total ED visits per capita in urban counties. Since our weighting approach uses population as the weights, rural counties would tend to be overrepresented in estimated values.
```

Comment:
Also maybe just "may be inaccurate but we don't know how"? The examples here are true if rural counties are reporting, but if rural counties aren't reporting anything at all, then we will be biased towards urban counties, I guess.

Reply:
I may have overthought this

Reply:
I think it's clearer than what I had before. I'm changing the wording a bit to make it clearer it's a hypothetical, and not something we have strong evidence for.